techio.yml

```diff
@@ -0,0 +1,19 @@
+title : Ruby
+
+# Table of content
+plan:
+  - title: Welcome # Set the page title
+    statement: markdowns/welcome.md # Set the file path to the page content
+#  - title: Another Page
+#    statement: markdowns/another-page.md
+
+
+# Settings for your programming projects and the associated Docker images
+projects:
+  ruby:
+    root: /ruby-project # Set the root path to your ruby project
+
+    # Set the docker image. This image runs JUnit tests using this syntax:
+    # @[Code Editor Title]({"stubs": ["EditorFile1.ruby", "EditorFile2.ruby", ...], "command": "ruby example.rb"})
+    # More details here: https://github.com/CodinGame/ruby-bundler-runner
```

Comment:
We will move soon all the code source in the techio organisation. You will have to rename this link though…